### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mstopa**

### CODE REVIEW

File: feb_rec.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+def feb_rec_item(n: int) -> int:
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
```
Comment (mstopa) [inline feb_rec.py]:
```
if n in (0, 1):
    return n
```

File: feb_rec.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    else:
+        result = feb_rec_item(n-1) + feb_rec_item(n-2)
```
Comment (mstopa) [inline feb_rec.py]: typo: feb -> fib

File: feb_rec.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        return result 
+    
+def feb_rec(m):
+    fibonacci_numbers = []
+    for i in range(m+1):
+        fibonacci_numbers.append(feb_rec_item(i))
+    return fibonacci_numbers
+
```
Comment (mstopa) [inline feb_rec.py]: please format code with pylint or black. incorrect number of empty newlines

File: fib_seq.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+def feb_seq_item(n: int) -> int:
+    fibonacci_numbers = []
+    one_step_back = 1
+    two_step_back = 0
+    for i in range(n+1):
+
```
Comment (mstopa) [inline fib_seq.py]: please use linter

File: fib_seq.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        
+        if i == 0:
+            item = 0
+        elif i == 1:
+            item = 1
```
Comment (mstopa) [inline fib_seq.py]: simplify with `if i in (0, 1)`

File: feb_rec.py (hunk 1 above)
Comment (mroszkow) [reply feb_rec.py]: Done

File: feb_rec.py (hunk 2 above)
Comment (mroszkow) [reply feb_rec.py]: Done

File: fib_seq.py (hunk 4 above)
Comment (mroszkow) [reply fib_seq.py]: Done

File: fib_seq.py (hunk 5 above)
Comment (mroszkow) [reply fib_seq.py]: Done

**Pull request merged**